presto-spi/src/main/java/com/facebook/presto/spi/connector/classloader/ClassLoaderSafeConnectorMetadata.java

```diff
@@ -30,6 +30,7 @@
 import com.facebook.presto.spi.SchemaTablePrefix;
 import com.facebook.presto.spi.classloader.ThreadContextClassLoader;
 import com.facebook.presto.spi.connector.ConnectorMetadata;
+import com.facebook.presto.spi.security.Privilege;
```

Comment:
I have `revokeTablePrivileges` implemented here. If you meant `grantTablePrivileges`, I have checked in that part of the code to the repo in this commit : https://github.com/prestodb/presto/commit/d9054f2238404e6082fefe476ae55dc2b9306793 as a separate PR.